packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -209,12 +216,30 @@ const NowPlayingDrawer = ({
   }, [trackId])
 
   const onNext = useCallback(() => {
-    setMediaKey(mediaKey => mediaKey + 1)
-  }, [setMediaKey])
+    if (track?.genre === Genre.PODCASTS) {
+      if (global.progress) {
+        const { currentTime } = global.progress
+        const newPosition = currentTime + SKIP_DURATION_SEC
+        dispatch(seek({ type: SEEK, seconds: Math.max(0, newPosition) }))
```

Comment:
What happens if the seconds exceed the length of the podcast?

Reply:
ah good catch. yea should be min with the track duration